packages/flutter/lib/src/services/platform_views.dart

```diff
@@ -74,6 +74,162 @@ class PlatformViewsService {
   }
 }
 
+/// Properties of an Android pointer.
+///
+/// A Dart version of Android's [MotionEvent.PointerProperties](https://developer.android.com/reference/android/view/MotionEvent.PointerProperties).
+class AndroidPointerProperties {
+  /// Creates an AndroidPointerProperties.
+  const AndroidPointerProperties({@required this.id, @required this.toolType});
+
+  /// See Android's [MotionEvent.PointerProperties#id](https://developer.android.com/reference/android/view/MotionEvent.PointerProperties.html#id).
+  final int id;
```

Comment:
might be clearer if there was a brief description of each of these before the link to the android docs.

Reply:
Added descriptions to most field (left out the ones where I feel there's no value I can add other than referencing the Android docs)